apps/dashboard/src/index.css

```diff
@@ -49,33 +90,253 @@
     --neutral-alpha-900: 231 100% 3% / 0.9;
     --neutral-alpha-950: 219 88% 3% / 0.95;
 
-    --accent: 0 0% 96.1%;
+    --blue-950: 228 70% 24%;
+    --blue-500: 227.94 100% 60%;
+    --blue-400: 222.12 100% 70.39%;
+    --blue-300: 219.81 100% 79.61%;
+    --blue-200: 220 100% 87.65%;
+    --blue-100: 221.43 100% 91.76%;
+    --blue-50: 222 100% 96.08%;
 
-    --destructive: 355 96% 60%;
-    --destructive-foreground: 0 0% 100%;
+    --blue-alpha-24: 228 100% 64% / 24%;
+    --blue-alpha-16: 228 100% 64% / 16%;
+    --blue-alpha-10: 228 100% 64% / 10%;
+
+    --orange-950: 20 70% 24%;
+    --orange-900: 20 71% 32%;
+    --orange-800: 20 70% 40%;
+    --orange-700: 20 69% 48%;
+    --orange-600: 20 80% 56%;
+    --orange-500: 20 100% 64%;
+    --orange-300: 20 100% 64% / 60%;
+    --orange-200: 20 100% 64% / 40%;
+    --orange-100: 20 100% 64% / 20%;
+    --orange-50: 20 100% 64% / 10%;
+
+    --orange-alpha-24: 20 100% 64% / 24%;
+    --orange-alpha-16: 20 100% 64% / 16%;
+    --orange-alpha-10: 20 100% 64% / 10%;
+
+    --red-950: 355 70% 24%;
+    --red-900: 355 71% 32%;
+    --red-800: 355 70% 40%;
+    --red-700: 355 70% 48%;
+    --red-600: 355 80% 56%;
+    --red-500: 355 96% 60%;
+    --red-400: 355 96% 60% / 80%;
+    --red-300: 355 96% 60% / 60%;
+    --red-200: 355 96% 60% / 40%;
+    --red-100: 355 96% 60% / 20%;
+    --red-50: 355 96% 60% / 10%;
+
+    --red-alpha-24: 355 96% 60% / 24%;
+    --red-alpha-16: 355 96% 60% / 16%;
+    --red-alpha-10: 355 96% 60% / 10%;
+
+    --green-950: 148 73% 16%;
+    --green-900: 148 64% 24%;
+    --green-800: 148 64% 28%;
+    --green-700: 148 72% 32%;
+    --green-600: 148 72% 40%;
+    --green-500: 148 72% 44%;
+    --green-400: 148 72% 44% / 80%;
+    --green-300: 148 72% 44% / 60%;
+    --green-200: 148 72% 44% / 40%;
+    --green-100: 148 72% 44% / 20%;
+    --green-50: 148 72% 44% / 10%;
+
+    --green-alpha-24: 148 72% 44% / 24%;
+    --green-alpha-16: 148 72% 44% / 16%;
+    --green-alpha-10: 148 72% 44% / 10%;
+
+    --yellow-950: 42 61% 24%;
+    --yellow-900: 42 64% 32%;
+    --yellow-800: 42 64% 40%;
+    --yellow-700: 42 64% 48%;
+    --yellow-600: 42 80% 50%;
+    --yellow-500: 42 92% 54%;
+    --yellow-400: 42 92% 54% / 80%;
+    --yellow-300: 42 92% 54% / 60%;
+    --yellow-200: 42 92% 54% / 40%;
+    --yellow-100: 42 92% 54% / 20%;
+    --yellow-50: 42 92% 54% / 10%;
+
+    --yellow-alpha-24: 42 92% 64% / 24%;
+    --yellow-alpha-16: 42 92% 64% / 16%;
+    --yellow-alpha-10: 42 92% 64% / 10%;
+
+    --purple-950: 258 64% 28%;
+    --purple-900: 258 64% 32%;
+    --purple-800: 258 64% 40%;
+    --purple-700: 258 64% 48%;
+    --purple-600: 256 72% 56%;
+    --purple-500: 256 88% 64%;
+    --purple-400: 256 88% 64% / 80%;
+    --purple-300: 256 88% 64% / 60%;
+    --purple-200: 256 88% 64% / 40%;
+    --purple-100: 256 88% 64% / 20%;
+    --purple-50: 256 88% 64% / 10%;
```

Comment:
Isn't it a bit weird that some colors have opacity and some not ?

Reply:
I tend to agree, @twentyone24 do you think we can make all of the opacity based? Or is it too much effort?

Reply:
If we can make it, then we don't need to define all the tokens in this CSS file. Instead, we will just have "base" colors and generate all the "semantic" ones in the tailwind config. Otherwise still we can simplify the opacity variables by defining them in the tw.